crates/types/src/logs/metadata.rs

```diff
@@ -152,11 +152,7 @@ impl ProviderConfiguration {
     pub fn from_configuration(configuration: &Configuration) -> Self {
         ProviderConfiguration::from((
             configuration.bifrost.default_provider,
-            configuration
-                .bifrost
-                .replicated_loglet
-                .default_log_replication
-                .clone(),
+            configuration.common.default_replication.clone(),
```

Comment:
Wondering whether we should remove the `ReplicatedLogletOptions::default_log_replication` in 1.4 and only deprecate its usage in 1.3. That way, we don't break people's configuration while giving them time to migrate to the new option before we remove the configuration option in 1.4. Here, the logic would then be, if `ReplicatedLogletOptions::default_log_replication` is set, use this value, if not, then use `CommonOptions::default_replication`.